### PR TITLE
Gt 111b

### PR DESCRIPTION
**Pull request by JakeTheGingScheuer**
Added google credentials to be used in kdp minio gateway. Values are passed in on set, then a json config is made dynamically and stored in the docker image upon creation. The json file is consumed by the minio chart by location.

### CODE REVIEW

File: docs/index.yaml [hunk 1]
```diff
@@ -2,9 +2,9 @@ apiVersion: v1
 entries:
   andi:
   - apiVersion: v1
-    created: 2019-11-21T10:56:05.248175-05:00
+    created: 2019-11-22T15:45:54.234216-05:00
```
Comment (JShefferPillar) [inline docs/index.yaml]: I think the created and digest ids should be reverted for andi, discovery-api, and forklift.

Keep kubernetes-data-platform

File: kubernetes-data-platform/templates/minio/deployment.yaml [hunk 2]
```diff
@@ -39,9 +39,19 @@ spec:
...
           - containerPort: {{ .Values.minio.service.port }}
             name: minio
+          volumeMounts:
+          - mountPath: /data/config.json
+            name: google-creds
+            subPath: config.json
+      volumes:
+        - name: google-creds
+          configMap:
+            name: creds
```
Comment (JShefferPillar) [inline kubernetes-data-platform/templates/minio/deployment.yaml]: name this google-credentials

File: kubernetes-data-platform/templates/minio/google-creds.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: creds
```
Comment (JShefferPillar) [inline kubernetes-data-platform/templates/minio/google-creds.yaml]: name this google-credentials

File: kubernetes-data-platform/values.yaml [hunk 4]
```diff
@@ -103,6 +103,13 @@ minio:
   gateway:
     enable: false
     type: azure
+  creds:
```
Comment (JShefferPillar) [inline kubernetes-data-platform/values.yaml]: name this google-credentials

File: platform/values.yaml [hunk 5]
```diff
@@ -82,6 +82,13 @@ kubernetes-data-platform:
     gateway:
       enable: false
       type: azure
+    creds:
```
Comment (JShefferPillar) [inline platform/values.yaml]: name this google-credentials

**Review by JShefferPillar — CHANGES_REQUESTED** [review]
Just a couple cleanup items

**Review by JakeTheGingScheuer — COMMENTED** [review]
changes were completed

**Pull request merged**